File: test_soal1.php
<?php 


// - buatlah program FizzBuzz dengan looping: 
// - Jika angka yang keluar adalah angka 5 maka output yang di hasilkan adalah "Fizz"
// - Jika angka yang keluar adalah angka 11 maka output yang di hasilkan adalah "Buzz"
// - Jika angka yang keluar adalah 5 dan 11 maka output yang di hasilkan adalah "FizzBuzz"

$data = [1, 2, 4, 3, 10, 11, 20, 5, 100, 200];
foreach ($data as $angka) {
    // Jika angka yang keluar adalah 5 dan 11 maka output yang di hasilkan adalah "FizzBuzz"
    if ($angka == 5 && $angka == 11) {
        echo "FizzBuzz" . "\n";
    // Jika angka yang keluar adalah angka 5 maka output yang di hasilkan adalah "Fizz"
    } else if ($angka == 5) {
        echo "Fizz" . "\n";
    // Jika angka yang keluar adalah angka 5 maka output yang di hasilkan adalah "Buzz"
    } else if ($angka == 11) {
        echo "Buzz" . "\n";
    } 
}

File: Test_soal1.java

package test;

import java.util.Scanner;

public class Test_soal1 {
    public static void main(String[] args) {
   for(int i = 1; i <= 200; i++) {           // count from 1 to 100
            if (((i % 5) == 0) && ((i % 11) == 0)) // Is it a multiple of 5 & 7?
                System.out.print("fizzbuzz");   
            else if ((i % 5) == 0)                // Is it a multiple of 5?
                System.out.print("fizz"); 
            else if ((i % 11) == 0)                // Is it a multiple of 7?
                System.out.print("buzz"); 
            else System.out.print(i);             // Not a multiple of 5 or 7
            System.out.print(" ");                
        }
        System.out.println( );
    }
}